src/models/adr-details.ts

```diff
@@ -9,24 +10,101 @@ export interface AdrDetails {
   brakeDeclarationsSeen?: boolean;
   brakeDeclarationIssuer?: string;
   brakeEndurance?: boolean;
-  weight?: string;
-  compatibilityGroupJ?: boolean;
+  weight?: number;
+  newCertificateRequested?: boolean;
```

Comment:
Is this required? It is not in the new updated NOP tables.

Reply:
Weight is listed as a number in ADR section, so I would presume that is required, it's how I've done it in Pydantic. CompatibilityGroupJ should still be listed though no?